mlir/lib/Conversion/ArithToEmitC/ArithToEmitC.cpp

```diff
@@ -112,6 +112,78 @@ class CmpIOpConversion : public OpConversionPattern<arith::CmpIOp> {
   }
 };
 
+template <typename ArithOp, bool needsUnsigned>
```

Comment:
What does `needsUnsigned` stand for? Does it mean the input should be interpreted as unsigned?

Reply:
`needsUnsigned` means that the cast op is to an unsigned type (non-sign-extending, guaranteed truncation prior to C++20). Perhaps `castToUnsigned` would be a better job :)